schemas/datePublished.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think that we then should use pubDate also for products since the product doesn't really start. It is released.